project2/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=ZCOOL+QingKe+HuangYou|Khand" rel="stylesheet">
+</head>
+<title>Card Specimen of Spider-Man</title>
+
+<body>
+    <div class="rapper">
+        <figure>
+            <img src="images/Spiderman.jpg" alt="spider-man with black background" width="200px" height="200px">
```

Comment:
2 notes here:
1. In general, I want you to control the height/width of an image via CSS, not inline.
2. Even for inline control, you don't use units, so it would just be `width="200" height="200"`

```suggestion
            <img src="images/Spiderman.jpg" alt="spider-man with black background">
```